src/track/beatutils.cpp

```diff
@@ -82,16 +82,15 @@ double BeatUtils::computeSampleMedian(QList<double> sortedItems) {
 }
 
 QList<double> BeatUtils::computeWindowedBpmsAndFrequencyHistogram(
-        const QVector<double> beats, const int windowSize, const int windowStep,
-        const int sampleRate, QMap<double, int>* frequencyHistogram) {
+        const QVector<double> beats, const int windowSize, const int windowStep, const int frameRate, QMap<double, int>* frequencyHistogram) {
     QList<double> averageBpmList;
     for (int i = windowSize; i < beats.size(); i += windowStep) {
         //get start and end sample of the beats
         double start_sample = beats.at(i - windowSize);
         double end_sample = beats.at(i);
 
         // Time needed to count a bar (4 beats)
-        double time = (end_sample - start_sample) / sampleRate;
+        double time = (end_sample - start_sample) / frameRate;
```

Comment:
We still use the term *sample rate* that always refers to samples per channel. The term *frame rate* is only used for video.